proxies/live/apiproxy/policies/AssignMessage.Errors.CatchAllTarget.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Errors.CatchAllTarget">
+    <Set>
+      <Payload contentType="application/json" variablePrefix="%" variableSuffix="#">
+      {
+        "resourceType": "OperationOutcome",
+        "meta": {
+          "profile": [
+            "https://fhir.hl7.org.uk/StructureDefinition/NHSDigital-OperationOutcome"
```

Comment:
Should be https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome

As per https://nhsd-confluence.digital.nhs.uk/display/WAYF/JSON+Examples#JSONExamples-ErrorResponse and https://nhsd-confluence.digital.nhs.uk/display/WAYF/FHIR+Profiles

Reply:
Fixed